core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -5,17 +5,23 @@ import { OperatorPanelComponent } from './operator-panel/operator-panel.componen
 import { PropertyEditorComponent } from './property-editor/property-editor.component';
 import { WorkflowEditorComponent } from './workflow-editor/workflow-editor.component';
 import { ResultPanelComponent } from './result-panel/result-panel.component';
+import { OperatorMetadataService } from '../service/operator-metadata/operator-metadata.service';
 
```

Comment:
Shouldn't there be a workspace html where you include operator label and operator panel html tags?